scripts/python/lib/argparse_gen.py

```diff
@@ -313,14 +332,22 @@ def _check_deploy(args, subparser):
             args.install_client_os == ABSENT and
             args.ssh_keyscan == ABSENT and
             args.gather_mac_addr == ABSENT and
-            args.lookup_interface_names == ABSENT and
             args.config_client_os == ABSENT and
             args.all == ABSENT):
         subparser.error(
             'one of the arguments --create-inventory --install-cobbler'
             ' --inv-add-ports-pxe --inv-add-ports-ipmi --download-os-images'
             ' --add-cobbler-distros --add-cobbler-systems --install-client-os'
-            ' --ssh-keyscan --gather-mac-addr --lookup-interface-names '
+            ' --ssh-keyscan --gather-mac-addr --config-client-os -a/--all is'
```

Comment:
' --ssh-keyscan --gather-mac-addr --config-client-os' should not be included in '_check_deploy' if they're moving to '_check_post_deploy'.